### ISSUE TITLE
Cascading str_replace(): a false-positive

### ISSUE THREAD

**Issue by kalessil** [kalessil/phpinspectionsea#399]
```php
    $a = '...';
    $b = str_replace('a', 'b', $a);
    $c = str_replace('c', 'd', $b);
    print "$a:$b:$c";
```

**Comment by tiagovb** [comment]
+1
